Library/Homebrew/cask/blacklist.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Cask
+  module Blacklist
+    def self.blacklisted_reason(name)
+      case name
+      when /^adobe\-.+/i
+        "Adobe casks were removed because they are too difficult to maintain."
+      when /audacity/i
+        "Audacity was removed because it is too difficult to download programmatically."
+      when /pharo/i
+        "Developers maintain their own tap."
```

Comment:
```suggestion
        "Pharo developers maintain their own tap."
```

(For consistency with the other messages)